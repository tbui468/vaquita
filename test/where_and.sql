create database sol;
open sol;

create table planets (id int8 key, name text, moons int8);
insert into planets (id, name, moons) values (1, "Mars", 2);
select * from planets where true and true;
select * from planets where true and false;
select * from planets where false and false;
select * from planets where false and true;

close sol;
drop database sol;
exit;
