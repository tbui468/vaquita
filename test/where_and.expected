created database sol
opened database sol
created table planets
inserted 1 record(s) into planets
id, name, moons
1, Mars, 2
id, name, moons
id, name, moons
id, name, moons
closed database sol
dropped database sol
